pyproject.toml

```diff
@@ -22,8 +22,14 @@ poetry-core = "^1.8.1"
 lightkube = "^0.15.0"
 lightkube-models = "^1.29.0.6"
 pydantic = "^1.10.14"
-psycopg2-binary = "^2.9.9"
-psycopg = {extras = ["binary"], version = "^3.1.17"}
+
+
+[tool.poetry.group.cdeps]
+optional = true
+
+[tool.poetry.group.cdeps.dependencies]
+psycopg2 = "^2.9.9"
+psycopg = {extras = ["c"], version = "^3.1.17"}
```

Comment:
If installing psycopg in the main group. It will be compiled at each unit test run and the unit tests won't be able to run on systems that do not have libpq installed.